python_modules/dagster-test/dagster_test/toys/external_execution/__init__.py

```diff
@@ -79,5 +80,5 @@ def number_sum(
         materialize(
             [number_x, number_y, number_sum],
             instance=instance,
-            resources={"ext": pipes_subprocess_client},
+            resources={"pipes_subprocess_client": pipes_subprocess_client},
```

Comment:
intentionally included?

Reply:
yea, used this example to test dagit as well  - needed this change to work